e2e/support/helpers/e2e-ui-elements-overflow-helpers.js

```diff
@@ -34,14 +34,10 @@ export const assertDescendantNotOverflowsContainer = (
     return;
   }
 
-  expect(descendantRect.bottom, `${message} bottom`).to.be.lte(
-    containerRect.bottom,
-  );
-  expect(descendantRect.top, `${message} top`).to.be.gte(containerRect.top);
-  expect(descendantRect.left, `${message} left`).to.be.gte(containerRect.left);
-  expect(descendantRect.right, `${message} right`).to.be.lte(
-    containerRect.right,
-  );
+  cy.wrap(descendantRect.bottom).should("be.lte", containerRect.bottom);
+  cy.wrap(descendantRect.top).should("be.gte", containerRect.top);
+  cy.wrap(descendantRect.left).should("be.gte", containerRect.left);
+  cy.wrap(descendantRect.right).should("be.lte", containerRect.right);
```

Comment:
Can you try [if you could keep using `expect` inside `should`](https://docs.cypress.io/api/commands/should#Assert-text-content-of-3-elements) to retain the error `message` for clarity?

I haven't tried this, but I'd expect something like this:

```suggestion
  cy.wrap(descendantRect).should(descendantRect => {
    expect(descendantRect.bottom, `${message} bottom`).to.be.lte(
      containerRect.bottom,
    );
    expect(descendantRect.top, `${message} top`).to.be.gte(containerRect.top);
    expect(descendantRect.left, `${message} left`).to.be.gte(containerRect.left);
    expect(descendantRect.right, `${message} right`).to.be.lte(
      containerRect.right,
    );
  })
```

Reply:
it works! ed9ccd049233d4647e7624e45470b444a8b4135e